### PR TITLE
Add some freeglut related help messages

### PR DESCRIPTION
**Pull request by hellwolf**
#103

### CODE REVIEW

File: play [hunk 1]
```diff
@@ -80,13 +80,23 @@ sudoku()            { runghc "$1"; }
...
     printf '%0.1s' "="{1..80};echo
     echo "= " "$1"
     printf '%0.1s' "="{1..80};echo
+    cat "$1"
 }
 gamecat()  { echo "$1" | cut -d/ -f1; }
 gamename() { basename "$1" .hs; }
+detectAndWarnAboutFreeglut() {
+    # NOTE: The detection logic requires gcc, without it will produce false alarm.
+    #       But can you imagine the life of people without gcc?
```
Comment (simonmichael) [inline play:93]: Yes, I think I can imagine linux systems which don't yet have dev tools like gcc installed. There's nothing worse than misleading error messages, so if we must use gcc here I think it's better to first check for that and give a more specific message.

**Review by simonmichael — CHANGES_REQUESTED** [review]
This is quite freeglut-specific, I assume there'll be other issues to warn about, but a good start.

**Comment by hellwolf** [comment]
> This is quite freeglut-specific, I assume there'll be other issues to warn about, but a good start.

Indeed, we could document some lib per-requisites, but it gets daunting and error prone.

FWIW, In case of Nix (~/.stack/config.yaml)

```
nix:
  enable: true
  packages:
    [ zlib.dev
    , zlib.out
    , libGL.dev
    , libGL.out
    , libGLU.dev
    , libGLU.out
    ]
```

I guess we could keep doing incremental improvements.

**Comment by simonmichael** [comment]
Thanks!

**Pull request merged**